### PR TITLE
Add support for container advisories

### PR DESCRIPTION
**Pull request by mprahl**

### CODE REVIEW

**Comment by yashvardhannanavati** [comment]
In the commit message, do you mean container advisory? If yes, could you please change the commit message

**Comment by mprahl** [comment]
Oops. I'll fix that.

**Comment by mprahl** [comment]
@yashvardhannanavati fixed

File: src/app/story/storyrow/storyrow.component.ts [hunk 1]
```diff
@@ -87,6 +87,7 @@ export class StoryRowComponent implements OnInit, AfterViewInit {
       case('KojiBuild'):
         return 'pficon-build';
       case('Advisory'):
+      case('ContainerAdvisory'):
         return 'pficon-security';
```
Comment (yashvardhannanavati) [inline src/app/story/storyrow/storyrow.component.ts]: I think a return statement for Advisory is missing.

Comment (mprahl) [reply src/app/story/storyrow/storyrow.component.ts]: This is correct. This is basically saying, if the value is `Advisory` or `ContainerAdvisory`, return `'pficon-security'`.

Comment (yashvardhannanavati) [reply src/app/story/storyrow/storyrow.component.ts]: cool

**Pull request merged**